Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/API/Beckn/Confirm.hs

```diff
@@ -83,10 +83,8 @@ confirm transporterId (SignatureAuthResult _ subscriber) reqV2 = withFlowHandler
             Just rideInfo' -> do
               fork "on_confirm/on_update" $ do
                 handle (errHandler dConfirmRes transporter (Just rideInfo'.driver)) $ do
-                  -- We assign driver post `on_confirm`, so removing `rideInfo` details fron `dConfirmRes`.
-                  let dConfirmRes' = dConfirmRes {DConfirm.rideInfo = Nothing}
-                  callOnConfirm dConfirmRes' msgId txnId bapId callbackUrl bppId bppUri city country
-                  void $ BP.sendRideAssignedUpdateToBAP dConfirmRes.booking rideInfo'.ride rideInfo'.driver rideInfo'.vehicle
+                  callOnConfirm dConfirmRes msgId txnId bapId callbackUrl bppId bppUri city country
+            -- void $ BP.sendRideAssignedUpdateToBAP dConfirmRes.booking rideInfo'.ride rideInfo'.driver rideInfo'.vehicle
```

Comment:
Can you add comment here ?
And while raising PR to main, Can you add the checks that we discussed on Friday, to check status updates only in forward direction.